base_layer/wallet_ffi/.venv/lib/python3.12/site-packages/_pytest/helpconfig.py

```
@@ -0,0 +1,283 @@
+        "e.g. `-o xfail_strict=True -o cache_dir=cache`.",
+    )
+
+
+@pytest.hookimpl(wrapper=True)
+def pytest_cmdline_parse() -> Generator[None, Config, Config]:
+    config = yield
+
+    if config.option.debug:
+        # --debug | --debug <file.log> was provided.
+        path = config.option.debug
+        debugfile = open(path, "w", encoding="utf-8")
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)